src/screens/OrganizationEvents/CreateEventModal.tsx

```
@@ -0,0 +1,589 @@
+          recurrenceInput = formatRecurrenceForApi(recurrence);
+        }
+
+        const input = {
+          name: formState.name,
+          description: formState.eventdescrip,
+          startAt: alldaychecked
+            ? dayjs(startDate)
+                .startOf('day')
+                .format('YYYY-MM-DDTHH:mm:ss.SSS[Z]')
+            : dayjs(startDate)
+                .hour(parseInt(startTimeParts[0]))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)